tools/internal_ci/linux/grpc_xds_bazel_test_in_docker.sh

```diff
@@ -45,12 +45,24 @@ touch "$TOOLS_DIR"/src/proto/grpc/testing/__init__.py
     "$PROTO_SOURCE_DIR"/messages.proto \
     "$PROTO_SOURCE_DIR"/empty.proto
 
-bazel build test/cpp/interop:xds_interop_client
+export GRPC_VERBOSITY=debug
+export GRPC_TRACE=xds_client,xds_resolver,cds_lb,xds_lb
 
-GRPC_VERBOSITY=debug GRPC_TRACE=xds_client,xds_resolver,cds_lb,xds_lb "$PYTHON" \
```

Comment:
This should still have verbose logging enabled.

More importantly, I'm not sure about combining Python and C++ in the same job. The runs for this test already take long enough that I don't think we want the same kokoro job to execute C++ and Python sequentially. Python should probably just have its own stand-alone xDS Kokoro job and corresponding script here on Github - at the least, C++ and Python should only be in the same job if they are configured to separately report status for Python and C++ for dashboards etc.

Reply:
I think I've split things up the way you want (along with a pending internal change). PTALA.

Reply:
C++ tests unrenamed as desired. PTALA.